openpype/pipeline/stagingdir.py

```diff
@@ -0,0 +1,146 @@
+import os
+import tempfile
+from openpype.lib import (
+    Logger,
+    filter_profiles
+)
+from openpype.pipeline import (
+    tempdir,
+    Anatomy
+)
+from openpype.settings import (
+    get_project_settings
+)
+
+
+TRANSIENT_DIR_TEMPLATE = "transient"
+
+
+def get_custom_staging_dir_info(project_name, host_name, family, task_name,
+                                task_type, subset_name,
+                                project_settings=None,
+                                anatomy=None, log=None):
```

Comment:
This returning some sort of "info" also sounds like this might expand over time. I'd say this could better return a `dict` with keys related to the info than a 2-tuple now which extends later breaking backwards compatibilities.

```python
return {
    "path": path,
    "persistent": persistent
}
```

Reply:
Yes I was considering this too. It had been inpremented.